app.py

```
@@ -58,36 +60,88 @@ async def login(user: UserLogin, db: Session = Depends(get_db)):
+    if data.content is not None:
+        temp_diary.content = data.content
+    if data.created_at is not None:
+        temp_diary.created_at = data.created_at
+        
+    db.commit()
+
+    return {"message": "수정 성공"}
+
+# 기록 삭제
+@app.delete("/api/temp_diary/delete")
+async def delete_temp_diary(temp_diary_id: int, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)